app/tests/unit/components/Layout/Footer.test.tsx

```diff
@@ -1,9 +1,10 @@
 import Footer from "components/Layout/Footer";
 import { render, screen } from "@testing-library/react";
+import footerLinks from "../../../../data/dashboardLinks/footerLinks";
 
 describe("The Footer Component", () => {
   it("Renders footer links", () => {
```

Comment:
Might be good to make this a "renders default footer links" test, and then have an additional test for the external footer